build.gradle

```diff
@@ -87,10 +88,10 @@ subprojects { subproject ->
 		log4jVersion = '1.2.17'
 		logbackVersion = '1.1.2'
 		mockitoVersion = '1.9.5'
-		rabbitmqVersion = project.hasProperty('rabbitmqVersion') ? project.rabbitmqVersion : '3.5.7'
+		rabbitmqVersion = project.hasProperty('rabbitmqVersion') ? project.rabbitmqVersion : '3.6.0'
 		rabbitmqHttpClientVersion = '1.0.0.RELEASE'
 
-		springVersion = project.hasProperty('springVersion') ? project.springVersion : '4.2.2.RELEASE'
+		springVersion = project.hasProperty('springVersion') ? project.springVersion : '4.2.4.RELEASE'
```

Comment:
I have no problem making 4.2.x the minimum required version if there's some compelling reason but I don't feel this is compelling.